### PR TITLE
connected mongodb and mongoose to backend

### PR DESCRIPTION
**Pull request by ncen5293**

### CODE REVIEW

File: server/routes/users.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const UserSchema = new Schema({
+  _id: ObjectId,
+  email: String,
+  password: String,
+  companies: []
+});
+
+const UserModel = mongoose.model('user', UserSchema);
+
```
Comment (aeciorc) [inline server/routes/users.js]: I'd rather those were not here. We want to have a folder just for models instead. You can remove these lines for now, as we just want to connect to the database

File: server/routes/users.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+const router = express.Router();
+const mongoose = require("mongoose");
+
+let databaseConnection = "Waiting for Database response...";
+
+router.get("/", (req, res, next) => {
+    res.send(databaseConnection);
+});
+
+mongoose.connect("mongodb://localhost:27017/mentions", { useNewUrlParser: true  });
```
Comment (aeciorc) [inline server/routes/users.js]: I'd rather the url is kept in the .env file

File: server/routes/users.js (hunk 1 above)
Comment (ncen5293) [reply server/routes/users.js]: alright, I removed them

File: server/routes/users.js (hunk 2 above)
Comment (ncen5293) [reply server/routes/users.js]: I changed to to use the env file

**Review by aeciorc — APPROVED** [review]
Looks good, let's get that merged

**Pull request merged**